### ISSUE TITLE
It seems that using FileZilla client to delete files on sdcard caused it died.

### ISSUE THREAD

**Issue by i1itione** [Laf111/WiiUFtpServer#26]
I don't have second sdcard to confirm, but this sdcard worked very well until I use FileZilla client to delete files on it. Also I saw a lots of I/O error messages on FileZilla client when I tried deleting files, maybe I shouldn't delete files via FTP, using a computer and card reader instead.

**Comment by ExpertStatue** [comment]
yeah this is done laf111 doesn't answer inject is also broken

**Comment by ExpertStatue** [comment]
you should try a diffrent ftp server

**Comment by i1itione** [comment]
> you should try a diffrent ftp server

Which FTP server is recommended on Wii U currently?

**Comment by i1itione** [comment]
> yeah this is done laf111 doesn't answer inject is also broken

May I ask a question, why an application would crash an sdcard technically?

**Comment by Laf111** [comment]
Sorry, i was on vaccation.
I'm surprised about that. 

Please, wait for the next release based on dynamic libs which will be much more stable (but without a channel version...)

Did you try to repair your sdcard (chkdsk /F on windows, i've forgoten for linux...) ?

**Comment by i1itione** [comment]
> Sorry, i was on vaccation. I'm surprised about that. Renaming is broken in the last version.
> 
> Please, wait for the next release based on dynamic libs which will be much more stable (but without a channel version...)
> 
> Did you try to repair your sdcard (chkdsk /F on windows, i've forgoten for linux...) ?

I tried to format this sdcard but it isn't able to be done even. (I use Windows 10)

It it just a 32G sdcard so that's ok, I wish this issue would not happen on anyone else again.

**Comment by Laf111** [comment]
Well, i'm very sorry for that...

How old was this card ? 

Files deletion should not lead in disk errors.
Were you transferring files in background when you try to delete your files ? 

Do you remember the IO error messages displayed on client side ? enable to access to the location ? 
(which is normal if your SDCard already died before deletion commands)

**Comment by i1itione** [comment]
I don't remember how old is this sd card, but it worked very well without any problem. What I did before it died is: 

1. run WiiUFtpServer 
2. backup important nand files (because I chose to mount Wii U's nand) 
3. the backup process spent about 1 hour (Maybe it caused by this Wii U has corrupted nand filesystem, it is hynix mlc.) 
4. After backup process finished, I remember I saw some errors on WiiUFtpServer, therefore I think the backup files might not useful and decide to detele them, so I use FileZilla client to delete those backup files directly, but deletion failed, finally I use card reader on PC and found this sd card became dead.

Sorry for my English.
